styles/components/_trading-card.scss

```diff
@@ -0,0 +1,32 @@
+.cmp-trading-card {
+  width: $trading-card-width;
+
+  &__hrs-bar {
+    margin-top: space("sm");
+    height: 0.5rem;
+    margin-bottom: 0;
+  }
+
+  &__name {
+    position: relative;
+    font-size: font-size("xl");
+    margin-bottom: 0;
```

Comment:
I think we need some margin above the name. It's sort of hard to tell in Figma, but it looks like there's about 1rem of space between the bar and the name.